include/fuse_lowlevel.h

```diff
@@ -613,6 +613,10 @@ struct fuse_lowlevel_ops {
 	 *
 	 * For every open call there will be exactly one release call.
 	 *
+	 * However, in the case of a forced unmount, the filesystem should
+	 * safely handle the possibility of not receiving remaining release
+	 * calls.
```

Comment:
..and omit the separate paragraph, because I don't think force unmount needs to be safely handled. Force implies unsafe, and handling this correctly may impose a big penalty in regular usage.